### PR TITLE
Add Future of AU eRequesting page

### PR DESCRIPTION
**Pull request by dt-r**
* added images
* added standard text with TBD for future of eRequesting requirements

### CODE REVIEW

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This guide is divided into several pages which are listed at the top of each pag
   - [AU eRequesting data for interoperability](auereqdi.html): This is a placeholder for when content becomes available.
   - [Relationship with other HL7 AU FHIR IGs](relationship.html): This is a placeholder for when content becomes available.
   - [AU Variance Statement](variance.html): This page documents variance from AU Base and AU Core.
-  - [Future of AU eRequesting](erequesting-future.html): This is a placeholder for when content becomes available.
+  - [Future of AU eRequesting](future.html): This is a placeholder for when content becomes available.
```
Comment (JayMurdoch) [inline input/pagecontent/index.md:30]: Suggest removing placeholder and updating in line with Core: "This page outlines the approach to developing AU eRequesting and yearly update cycle."

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Hi Danielle,
One minor update suggested. Thanks

File: input/pagecontent/index.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/index.md]: changed.

**Review by JayMurdoch — APPROVED** [review]
Change reviewed, no further issues.

**Pull request merged**